### PR TITLE
Create new unittests (one test), nose to execute, and add code coverage

### PR DESCRIPTION
**Pull request by kaitj91**
This PR does the following:
* Create new unittest folder and adds a single unit test using `unittest`. 
* Updates `tox.ini` to call `nosetests` on the new unittests folder.
* Provides code coverage over the newly added unittest

The new unittest added gave an overall test coverage of 35%.

### CODE REVIEW

File: sqlalchemy_jsonapi/unittests/main.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""Must set up flask app for sqlalchmey_jsonapi unittesting."""
+
+from flask import Flask
```
Comment (rockstar) [inline sqlalchemy_jsonapi/unittests/main.py]: Is `sqlalchemy_jsonapi` actually using flask? Is this even necessary?

(If yes, I guess our first goal is to make it...not?)

**Comment by deryckh** [comment]
We need to get tests passing first before I can look at the patch.

**Comment by kaitj91** [comment]
I forgot to update TravisCI. Will add tests to that and get those running. I only updated the tox.ini file.

**Comment by kaitj91** [comment]
After talking with deryckh I will make these unittests using the sqlalchemy db and not using the flask app, since the test is only testing the serializer which only needs a db.session. I will close this PR and put a new one with changes discussed.

**Pull request closed without merging**